src/NUnitFramework/framework/Internal/TestExecutionContext.cs

```diff
@@ -167,7 +167,7 @@ public TestExecutionContext( TestExecutionContext other )
         [ThreadStatic]
         private static TestExecutionContext current;
 #elif NETCF
-        private static TestExecutionContext current;
+        private static LocalDataStoreSlot slotContext = Thread.AllocateDataSlot();
```

Comment:
It is.  CurrentContext is the property. and it encapsulates the various methodologies.  current is only used by CurrrentContext, and only in the silverlight and portable builds/